packages/inventory/src/Stocktake/api/api.ts

```diff
@@ -134,23 +134,24 @@ export const getStocktakeQueries = (sdk: Sdk, storeId: string) => ({
     patch: RecordPatch<StocktakeFragment>
   ): Promise<UpdateStocktakeInput> => {
     const input = stocktakeParser.toUpdate(patch);
-    const result = await sdk.updateStocktake({ input, storeId });
+    const result = (await sdk.updateStocktake({ input, storeId })) || {};
```

Comment:
and then there's a bunch of these.. this is because if we get `Forbidden` then there's no data object returned, so this part blows up. We don't want the destructuring error to be toasted, just the 'Unable to create requisition' etc. So.. just being careful with the mutations, to not throw an error and this allows a better error state to be shown.